pkgs/os-specific/linux/minimal-bootstrap/mes/default.nix

```diff
@@ -27,12 +19,113 @@ let
     #undef SYSTEM_LIBC
     #define MES_VERSION "${version}"
   '';
+
+  # Maintenance note:
+  # Build steps have been adapted from build-aux/bootstrap.sh.in
+  # as well as the live-bootstrap project
+  # https://github.com/fosslinux/live-bootstrap/blob/1bc4296091c51f53a5598050c8956d16e945b0f5/sysa/mes-0.24.2/mes-0.24.2.kaem
+
+  # Maintenance note: list of source files derived from build-aux/configure-lib.sh
```

Comment:
no way to use the upstream scripts?

Reply:
oh you're totally right! i will have a play with it but it shouldn't be too complicated after all